BNA/src/Math/Matrix4x4.bf

```diff
@@ -241,5 +241,75 @@ namespace BNA.Math
 				(vec.x * matrix.M13) + (vec.y * matrix.M23) + (vec.z * matrix.M33) + (vec.w * matrix.M43),
 				(vec.x * matrix.M14) + (vec.y * matrix.M24) + (vec.z * matrix.M34) + (vec.w * matrix.M44));
 		}
+
+		public static void CreateRotationX(float radians, out Matrix4x4 result)
```

Comment:
Actually and for that matter, as far as style is concerned, I'd like the API to be consistent in terms of rotation-related methods taking degrees rather than radians.